src/transaction.rs

```diff
@@ -178,19 +183,35 @@ pub fn handle_transaction(
 
     let res: JSTxSaplingData = JSTxSaplingData {
         decrypted_notes: ser_comp_note,
+        decrypted_new_notes: ser_new_comp_note,
         nullifiers: ser_nullifiers,
         commitment_tree: hex::encode(buff),
     };
     Ok(serde_wasm_bindgen::to_value(&res).map_err(|_| "Cannot serialize tx output")?)
 }
 
+pub fn serialize_comp_note(
+    comp_note: Vec<(Note, IncrementalWitness<Node>)>,
+) -> Result<Vec<(Note, String)>, Box<dyn Error>> {
+    let mut ser_comp_note: Vec<(Note, String)> = vec![];
+    for (note, witness) in comp_note.iter() {
+        let mut buff = Vec::new();
+        witness
+            .write(&mut buff)
+            .map_err(|_| "Cannot write witness to buffer")?;
+        ser_comp_note.push((note.clone(), hex::encode(&buff)));
```

Comment:
Remove this clone, not sure if this is intended to be done in #93

Reply:
yes I will rebase this PR or #93 depending on which gets merged first